### ISSUE TITLE
[Question||Bug] Scrape suddenly stops with no more data, while there's still more data

### ISSUE THREAD

**Issue by CoreJa** [haccer/twint#1278]
# Issue Template
Please use this template!

### Initial Check
> If the issue is a request please specify that it is a request in the title (Example: [REQUEST] more features). If this is a question regarding 'twint' please specify that it's a question in the title (Example: [QUESTION] What is x?). Please **only** submit issues related to 'twint'. Thanks.

>Make sure you've checked the following:

- [x] Python version is 3.6;
- [x] Updated Twint with `pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint`;
- [x] I have searched the issues and there are no duplicates of this issue/question/request.

Actually there is a duplicates #1274, but the main issue is not the same.

### Command Ran
>Please provide the _exact_ command ran including the username/search/code so I may reproduce the issue.

```python
import twint

c = twint.Config()

c.Search = "covid flight"
c.Since = "2019-01-01"
c.Until = "2021-07-22"
c.Lang = "en"
# c.Location = True
c.Show_hashtags = True
c.Count = True
c.Store_csv = True
c.Output = "result_20210721.csv"

twint.run.Search(c)
```

### Description of Issue
>Please use **as much detail as possible.**

The key issue is scaper date. On this exact date(2021-07-21), twint will just stop and say no more data. Guess there's some broken tweets that twint can't handle? 

I just wonder how this would happen, is there a debug mode? How am I suppose to skip those broken data and continue?

By the way, I uncomment this line. Otherwise it will only fetch a few lines and stop.

https://github.com/twintproject/twint/blob/e7c8a0c764f6879188e5c21e25fb6f1f856a7221/twint/url.py#L92

### Environment Details
>Using Windows, Linux? What OS version? Running this in Anaconda? Jupyter Notebook? Terminal?

OS: Manjaro 21.1.3 Pahvo
Kernel: x86_64 Linux 5.13.13-1-MANJARO
Running with python 3.6

**Comment by DavidPerea** [comment]
I am also having the same problem. I work with Command Prompt (**CMD**), where I indicate my command: `twint -u gofundme`

but it only allows me to extract the tweets until September 15th. How can I solve that?

**Comment by medo0070** [comment]
I am facing the same problem. Anything can help?

**Comment by CoreJa** [comment]
I had to skip a few data as a work around.
Just skip those data.

**Comment by medo0070** [comment]
> I had to skip a few data as a work around.
> Just skip those data.

Do you mean that the problem still exists without any solution?

**Comment by CoreJa** [comment]
> > I had to skip a few data as a work around.
> > Just skip those data.
> 
> Do you mean that the problem still exists without any solution?

Right. For me I'll just have to skip that data on that specific date.

> The key issue is scaper date. On this exact date(2021-07-21), twint will just stop and say no more data. Guess there's some broken tweets that twint can't handle?

**Comment by medo0070** [comment]
> > > I had to skip a few data as a work around.
> > > Just skip those data.
> > 
> > 
> > Do you mean that the problem still exists without any solution?
> 
> Right. For me I'll just have to skip that data on that specific date.
> 
> > The key issue is scaper date. On this exact date(2021-07-21), twint will just stop and say no more data. Guess there's some broken tweets that twint can't handle?

I will try this issue too. Thanks for sharing .

**Comment by DavidPerea** [comment]
> > > > Tuve que omitir algunos datos como solución temporal. 
> > > > Simplemente omita esos datos.
> > > 
> > > 
> > > ¿Quiere decir que el problema sigue existiendo sin solución?
> > 
> > 
> > Derecha. Para mí, tendré que omitir esos datos en esa fecha específica.
> > > La cuestión clave es la fecha del escapulario. En esta fecha exacta (2021-07-21), twint simplemente se detendrá y no dirá más datos. ¿Supongo que hay algunos tweets rotos que twint no puede manejar?
> 
> También intentaré este problema. Gracias por compartir .

I have tried testing various dates but when I try to extract tweets after mid-September it tells me the following:

[!] No more data! Scraping will stop now.
found 0 deleted tweets in this search.

When if there are more previous tweets. Why does this happen?

**Comment by bensilver95** [comment]
I'm having the same issue!

**Comment by bensilver95** [comment]
Follow up:

The solution that @klojohn suggested in #1266 worked for me on a mac, but not on Linux (which is strange, since they said it worked for them on Linux). I have not tried it on Windows.

**Comment by DavidPerea** [comment]
> Hacer un seguimiento:
> 
> La solución que @klojohn sugirió en el [# 1266](https://github.com/twintproject/twint/issues/1266) funcionó para mí en una Mac, pero no en Linux (lo cual es extraño, ya que dijeron que les funcionó en Linux). No lo he probado en Windows.

If you try it with Windows, please can you indicate if it has worked for you. Thanks!

**Comment by christineeeeee** [comment]
having the same issue here as well

**Comment by 0xLukeSec** [comment]
Having the same issue
